src/documentation/runners/gearpump.md

```diff
@@ -0,0 +1,133 @@
+* Fault-tolerance with exactly-once processing guarantees
+* Application hot re-deployment
+
+The [Beam Capability Matrix]({{ site.baseurl }}/documentation/runners/capability-matrix/) documents the currently supported capabilities of the Gearpump Runner.
+
+## Writing Beam Pipeline with Gearpump Runner
+To use the Gearpump Runner in a distributed mode, you have to setup a Gearpump cluster first by following the Gearpump [setup quickstart](http://gearpump.apache.org/releases/latest/deployment/deployment-standalone/index.html).
+
+Suppose you are writing a Beam pipeline, you can add a dependency on the latest version of the Gearpump runner by adding to your pom.xml to enable Gearpump runner.
+And your Beam application should also pack Beam SDK explicitly and here is a snippet of example pom.xml:
+```java
+<dependencies>
+  <dependency>
+    <groupId>org.apache.beam</groupId>
+    <artifactId>beam-runners-gearpump</artifactId>
+    <version>{{ site.release_latest }}</version>
+  </dependency>
+
+  <dependency>
+    <groupId>org.apache.gearpump</groupId>
+    <artifactId>gearpump-streaming_2.11</artifactId>
+    <version>${gearpump.version}</version>
+    <scope>provided</scope>
+  </dependency>
+
+  <dependency>
+    <groupId>org.apache.gearpump</groupId>
+    <artifactId>gearpump-core_2.11</artifactId>
+    <version>${gearpump.version}</version>
+    <scope>provided</scope>
+  </dependency>
+
+  <dependency>
+    <groupId>org.apache.beam</groupId>
+    <artifactId>beam-sdks-java-core</artifactId>
+    <version>{{ site.release_latest }}</version>
+  </dependency>
+</dependencies>
+
+<build>
+  <plugins>
+    <plugin>
+      <groupId>org.apache.maven.plugins</groupId>
+      <artifactId>maven-shade-plugin</artifactId>
+      <configuration>
+        <createDependencyReducedPom>false</createDependencyReducedPom>
+        <filters>
+          <filter>
+            <artifact>*:*</artifact>
+            <excludes>
+              <exclude>META-INF/*.SF</exclude>
+              <exclude>META-INF/*.DSA</exclude>
+              <exclude>META-INF/*.RSA</exclude>
+            </excludes>
+          </filter>
+        </filters>
+      </configuration>
+      <executions>
+        <execution>
+          <phase>package</phase>
+          <goals>
+            <goal>shade</goal>
+          </goals>
+          <configuration>
+            <shadedArtifactAttached>true</shadedArtifactAttached>
+            <shadedClassifierName>shaded</shadedClassifierName>
+          </configuration>
+        </execution>
+      </executions>
+    </plugin>
+  </plugins>
+</build>
+```
+
+After running <code>mvn package</code>, run <code>ls target</code> and you should see your application jar like:
+```
+{your_application}-{version}-shaded.jar
+```
+
+## Executing the pipeline on a Gearpump cluster
+To run against a Gearpump cluster simply run:
+```
+gear app -jar /path/to/{your_application}-{version}-shaded.jar com.beam.examples.BeamPipeline --runner=GearpumpRunner ...
```

Comment:
This is OK and matches the status of other runners, but please file JIRAs like https://issues.apache.org/jira/browse/BEAM-1603 and https://issues.apache.org/jira/browse/BEAM-1631 for executing this without the user having the extra step of building the shaded jar.